.github/workflows/build-bottles.yml

```diff
@@ -0,0 +1,25 @@
+name: Build homebrew bottles
+on:
+  # Run on master
+  push:
+    branches: [master]
```

Comment:
From the Github Actions docs it seems it's possible to just tie this action to `release` directly? Even though it's conditional on relevant paths, we sometimes go a long way committing to master without making a release (and, consequently, without need to build bottles), and also we touch the formulae _after_ each release too, to put in the hashes. Duplicating the runs seems unnecessary.

I'm not sure which `release` activity type would be needed here, but we need the bottles to build on both full releases and prereleases _created from_ the auto-release, but not on _each_ auto-release. If that's not possible, then I guess these paths are the next best alternative.

Reply:
Github has draft releases which are only visible to repository maintainers, we can change `autorelease.sh` script to create a draft release and set up the action to run when the release is published (either as a pre-release or as a full release), so that it will only run for releases which you publish manually. Maybe we can even attach the bottles as assets automatically in this case. The only issue is that you'll have to wait 4 hours for the bottles to build after you make a release.

Another options are probably to run the workflow on tags of the form `v*`, or to trigger the workflow manually when you need it.

Reply:
Running on `v*` tags sounds good to me for now.

Reply:
Ok, I updated it to run on tags